src/reanimated2/threads.ts

```diff
@@ -9,7 +9,15 @@ import {
 const IS_JEST = isJest();
 const IS_NATIVE = !shouldBeUseWeb();
 
-let _runOnUIQueue: Array<[ComplexWorkletFunction<any[], any>, any[]]> = [];
+/**
+ * An array of [worklet, args] pairs.
+ * */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let _runOnUIQueue: Array<[WorkletFunction<any, unknown>, any[]]> = [];
+// TODO TYPESCRIPT
+// above line should be:
+// let _runOnUIQueue: Array<[WorkletFunction<any[], unknown>, any[]]> = [];
+// but for some reason TypeScript gives CLI error on that
```

Comment:
Give the following change at line (more or less) 114
``` diff
-    _runOnUIQueue.push([worklet, args]);
+    _runOnUIQueue.push([worklet as WorkletFunction<any, unknown>, args]);

```
And there shouldn't be any error with the valid type

Reply:
I made this a bit different (without `any` :pepega:)